examples/two-column-layout/src/app.tsx

```diff
@@ -0,0 +1,36 @@
+/** @jsx h */
+import { autocomplete } from '@algolia/autocomplete-js';
+import { h, render } from 'preact';
+
+import { querySuggestionsPlugin } from './plugins/querySuggestionsPlugin';
+import { recentSearchesPlugin } from './plugins/recentSearchesPlugin';
+
+import '@algolia/autocomplete-theme-classic';
+
+autocomplete({
+  container: '#autocomplete',
+  placeholder: 'Search products, articles, and FAQs',
+  debug: true,
```

Comment:
We should probably remove this before merging.

Reply:
This feature is very error-prone. This is unrelated to this PR but I'd like to discuss its removal.

Reply:
Good catch! I'll remove the `debug` option, thanks!